src/Behat/GuzzleExtension/Context/GuzzleContext.php

```diff
@@ -268,6 +268,30 @@ public function iGetAnUnsuccessfulResponseWithAStatusCodeOf($code)
     }
 
     /**
+     * Checks response body matches the following value
+     *
+     * Example: The the response body matches the following:
+     *   """
+     *   Id,Name,Age,Comment
+     *   1,"Richard Saunders",33,"some comment"
+     *   2,"Dave Nash",,"another comment"
+     *   3,{stored[person][name]},{stored[person][age]},{stored[person][comment]}
+     *   """
+     *
+     * @Then the response body matches the following:
+     *
+     * @param PyStringNode $body
+     * @throws ClientErrorResponseException
```

Comment:
If there is no `throw` keyword in the method, do not specify a `@throws` directive in the docblock. Please remove.